src/code/z_player_lib.c

```diff
@@ -1361,7 +1364,7 @@ bool func_80123F14(PlayState* play) {
 }
 
 s32 func_80123F2C(PlayState* play, s32 ammo) {
```

Comment:
I'll leave it up to you, but can probably name this function something like `Player_SetBButtonAmmo`, and change instances to use decimal.